app/src/main/res/layout/login_fragment.xml

```diff
@@ -36,6 +36,9 @@
 
                     <android.support.v7.widget.AppCompatAutoCompleteTextView
                         android:id="@+id/email_dropdown"
+                        android:drawableLeft="@drawable/ic_account_circle"
+                        android:drawableStart="@drawable/ic_account_circle"
```

Comment:
You should add these attributes in textInputLayout instead.

Reply:
@Masquerade0097 Please guide me for the build failing.